apps/src/templates/studioHomepages/Incubator.jsx

```diff
@@ -74,6 +74,22 @@ class Incubator extends Component {
               },
             ]}
           />
+          <TwoColumnActionBlock
+            imageUrl="shared/images/teacher-announcement/incubator-genai-graphic.png"
+            subHeading={i18n.incubator_genai_title()}
+            description={<span>{i18n.incubator_genai_desc()}</span>}
+            marginBottom="20px"
+            buttons={[
+              {
+                url: 'https://studio.code.org/courses/exploring-gen-ai-2024?section_id=5618725',
+                text: i18n.incubator_genai_course_button(),
+              },
+              {
+                url: 'https://code.org/curriculum/generative-ai',
```

Comment:
We should be able to use the `pegasus()` helper here to avoid including the domain in the string, which also has the advantage that it will stay on the current environment.

Reply:
I was just testing the button after removing the code.org domain locally and it returned a 404; since this is a webpage created by marketing, is it possible that it's only specifically on the main code.org environment?

For example, https://studio.code.org/curriculum/generative-ai leads to a 404 as well even though https://code.org/curriculum/generative-ai works on the live site.

Reply:
NVM, Cassie helped explain to me about the pegasus() helper, removed the absolute url